### ISSUE TITLE
Plugins tooltip on dashboards with only one Atomic site is misplaced

### ISSUE THREAD

**Issue by msilbers** [Automattic/wp-calypso#32821]
#### Steps to reproduce
1. Log into an account with only one site which is on the Business or eCommerce plan with at least one plugin installed (when testing, you can add Hello Dolly)
2. In Calypso, scroll down to Add a Site button

#### What I expected
For the manage plugins tooltip to point to Plugins, which is now under Tools, or possibly to wp-admin (not 100% sure). Also, for the Add a Site text to not be blocked.

#### What happened instead
The plugins tooltip points to Add a Site button.

#### Browser / OS version
First noticed on Chrome Version 73.0.3683.103, other Happiness Engineers confirmed they see it as well.

#### Screenshot / Video
<img width="474" alt="tooltipplugin" src="https://user-images.githubusercontent.com/31680407/57198364-d34a5e80-6f3f-11e9-8b64-c600d1ea43ea.png">

#### Context / Source

This was not a user report, but noticed in multiple sites while assisting users, since the nav drawer layout was changed.

**Comment by davemart-in** [comment]
@msilbers would you by chance be able to give me access to an Atomic site? I don't currently have access to any.

**Comment by alaczek** [comment]
Noticed this as well and adding "Fix the flows" tag.

For me this tooltip appeared immediately after installing the first plugin on a site. Initially I thought this had to do with screen resolution, but I got the same result from testing on 13" macbook, and on external screen. On my macbook scrolling to the bottom did not reveal the whole tooltip.
* 13" macbook
![image](https://user-images.githubusercontent.com/4550351/57667615-8fdea700-7647-11e9-92c7-73844e9270da.png)
* external screen
![image](https://user-images.githubusercontent.com/4550351/57670113-25caff80-7651-11e9-9ba7-dfabcea11c45.png)

So to summarise the two issues I saw:
* the tooltip appears too far to the left, obscuring the "WP Admin" text
* it's cut off at the bottom, regardless of resolution

> In Calypso, scroll down to Add a Site button
I did not see that, but maybe because I clicked the "Got it" button, before trying to navigate around the site?

@davemart-in in my testing this shows up only when you add a plugin for the first time, so your best bet to see it would be to set up a new site with Business plan and install a plugin. Ping me if you ever need more credits.

**Comment by davemart-in** [comment]
Thanks @alaczek! I think I'm actually just going to remove this particular tooltip. These tooltips use pixel level adjustments which are always going to prove problematic when added to a nav bar that is constantly changing.
